### PR TITLE
Retire Ebrc-based study access / data restriction

### PR DESCRIPTION
**Pull request by jtlong3rd**
Depends on https://github.com/VEuPathDB/EbrcWebsiteCommon/pull/83

This PR excises the versions of the study access controller, hooks, and components that live in `EbrcWebsiteCommon`.

### CODE REVIEW

File: Site/package.json [hunk 1]
```diff
@@ -17,5 +17,10 @@
     "react-dom": "16",
     "@types/react": "16",
     "@types/react-dom": "16"
+  },
+  "devDependencies": {
```
Comment (jtlong3rd) [inline Site/package.json:21]: For some reason, "[borrowing](https://github.com/VEuPathDB/ClinEpiWebsite/pull/11#issue-1044731713)" @veupathdb/study-data-access from `EbrcWebsiteCommon` wasn't working for me - when running `wb site`, I was getting various errors of the form

```
TS4090: Conflicting definitions for 'zzz' found at 'xxx' and 'yyy'. Consider installing a specific version of this library to resolve the conflict.
```

(`zzz` being `@types/react`, `@types/jquery`, and `@types/jqueryui`.)

Adding `@veupathdb/study-data-access` as a dev dependency, along with the "usual" dev dependencies of `"bubleify": "^2.0.0"` and `"ify-loader": "^1.1.0"` resolved the error.

@dmfalke - do you see any issues with this approach, or better yet, have a cleaner one in mind?

Comment (dmfalke) [reply Site/package.json:21]: That's odd. Is there a possibility you are running a custom version of node? What does `which node` return for you?

My only concern is that this version could get out-of-sync with the version used by EbrcWebsiteCommon.

**Review by dmfalke — APPROVED** [review]
This all looks good. Please see my inline comment and let me know before you merge. Thanks!

**Comment by jtlong3rd** [comment]
> This all looks good. Please see my inline comment and let me know before you merge. Thanks!

Hmm... `/usr/bin/node`

**Pull request merged**